apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-detail-page.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="container-fluid pl-xl-5 pr-xl-5 vo-theme">
   <perun-web-apps-back-button>
   </perun-web-apps-back-button>
-  <mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
+<!--  <mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>-->
```

Comment:
```suggestion
<mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
  <div *ngIf="!loading && vo !== undefined">
    <div class="d-flex page-title-headtitle">
      <mat-icon
        matTooltip="{{'VO_DETAIL.ENTITY' | translate}}"
        [ngStyle]="{'color':'black'}"
        svgIcon='perun-vo-black'
        class="perun-icon perun-icon-detail">
      </mat-icon>
      <div class="page-title-block">
        <div class="page-title-headline d-flex align-items-center">
            <a class="vo-link" [routerLink]="['/organizations/', vo.id]">
              {{vo.name}}
            </a>
            <span class="text-muted">
            &nbsp;#{{vo.id}}
            </span>
          <button
            *ngIf="editAuth"
            mat-icon-button
            aria-label="Edit Vo"
            (click)="editVo()">
            <mat-icon>edit</mat-icon>
          </button>
        </div>
        <span class="mt-1 entity-info">
          {{'VO_DETAIL.ENTITY' | translate}},
          {{'VO_DETAIL.SHORT_NAME' | translate}}: {{vo.shortName}}
        </span>
      </div>
    </div>
    <app-animated-router-outlet></app-animated-router-outlet>
  </div>
```
I thinked a little bit about it and this is the best solution according to me
the code above is vo-detail page, the spinner in vo-overview should stay as it is, 
use this solution in every entity

Reply:
I agree, this solution seems to be the best. I fixed it also at member-detail page.